src/components/dropdown/dropdown.js

```diff
@@ -663,11 +675,14 @@ class Dropdown extends React.Component {
     }
 
     content.push(
-      <div { ...this.listBlockProps } ref={ (node) => { this.listBlock = node; } }>
-        { this.listHTML }
-      </div>
+      <Portal onReposition={ () => { this.positionOptions(); } }>
+        <div { ...this.listBlockProps } ref={ (node) => { this.listBlock = node; } }>
+            { this.listHTML }
+        </div>
+      </Portal>
     );
 
+    this.positionOptions();
```

Comment:
should we trigger this when the list opens rather than during a method that gets called on render? (means it would get called less)

Reply:
maybe `onFocus`?

Reply:
done